api.json

```
@@ -471,13 +551,41 @@
           }
         }
       },
+      "paymentByMemberId": {
```

Lanjutan dari comment di atas, yang disini kalau diubah jadi `Payment` aja gimana ?

```suggestion
      "Payment": {
```